README.md

```diff
@@ -17,37 +17,39 @@ The official SurrealDB library for Golang.
 go get github.com/surrealdb/surrealdb.go
 ```
 
-### Usage
+### Quick Start
 
 ```go
 package main
-
 import (
-    "fmt"
-    "github.com/surrealdb/surrealdb.go"
+	"github.com/surrealdb/surrealdb.go"
+
 )
 
+type User struct {
+	ID      string `json:"id,omitempty"`
+	Name    string `json:"name"`
+	Surname string `json:"surname"`
+}
+
 func main() {
-	// Connect to SurrealDB
 	db, err := surrealdb.New("ws://localhost:8000/rpc")
 	if err != nil {
 		panic(err)
 	}
 
-	// Sign in
-	if _, err = db.Signin(map[string]string{
```

Comment:
Please could you revert this to what it was before? We do want the comment for clarity, and we do want to be more specific with types if we can

Reply:
![Screenshot 2023-11-19 at 23 17 56](https://github.com/surrealdb/surrealdb.go/assets/16410361/366d67fa-2a2f-48c5-984b-2258112e65f7)

Makes sense!

Reply:
Ideally the fields would be known. In case people make mistakes like `username` instead of `user`, or `password` instead of `pass` or not be aware of the other fields. Overall, the parameter needs to change.